setup.py

```diff
@@ -3,11 +3,8 @@
 
 _version_file = 'hooked/version.py'
 
-if sys.version_info.major < 3:
-    execfile(_version_file)
-else:
-    with open(_version_file) as vf:
-        exec(vf.read())
+with open(_version_file) as vf:
+    exec(vf.read())
```

Comment:
@jonprindiville the effect is different when you use `exec` within a function, then `local` vs `global` scope rules are different for py2 and py3 (at least that's my interpretation of that SO post)